packages/carbon-web-components/src/components/button/button-story.ts

```diff
@@ -9,177 +9,320 @@
-      ${Add16({ slot: 'icon' })}
+      type="ghost"
+      ?disabled="${disabled}"
+      href="${ifDefined(href)}"
+      size="${ifDefined(size)}"
+      @click=${onClick}
+      tooltip-text="${tooltipText}"
+      tooltip-alignment="${tooltipAlignment}"
+      tooltip-position="${tooltipPosition}">
+      Ghost
     </cds-btn>
   `;
 };
 
-icon.parameters = Default.parameters;
-
-export const textAndIcon = (args) => {
-  const { kind, disabled, size, href, iconLayout, isExpressive, onClick } =
-    args?.[`${prefix}-btn`] ?? {};
+export const IconButton = (args) => {
+  const {
+    disabled,
+    href,
+    size,
+    tooltipAlignment,
+    tooltipPosition,
+    tooltipText,
+    type,
+    onClick,
+  } = args?.[`${prefix}-btn`] ?? {};
   return html`
     <cds-btn
-      kind=${ifDefined(kind)}
-      ?disabled=${disabled}
-      icon-layout="${ifDefined(iconLayout)}"
-      ?isExpressive="${isExpressive}"
-      size=${ifDefined(size)}
-      href=${ifDefined(href || undefined)}
-      @click=${onClick}>
-      Button ${Add16({ slot: 'icon' })}
+      ?disabled="${disabled}"
+      href="${ifDefined(href)}"
+      type="${ifDefined(type)}"
+      size="${ifDefined(size)}"
+      @click=${onClick}
+      tooltip-text="${tooltipText}"
+      tooltip-alignment="${tooltipAlignment}"
+      tooltip-position="${tooltipPosition}">
+      ${Add16({ slot: 'icon' })}
     </cds-btn>
   `;
 };
 
-textAndIcon.storyName = 'Text and icon';
-
-textAndIcon.parameters = {
+IconButton.parameters = {
   knobs: {
     [`${prefix}-btn`]: () => ({
-      iconLayout: select('Icon layout (icon-layout)', iconLayouts, null),
-      kind: select('Button kind (kind)', kinds, BUTTON_KIND.PRIMARY),
+      type: select('Button type (type)', types, BUTTON_TYPE.PRIMARY),
+      tooltipAlignment: select(
+        'Tooltip alignment',
+        alignmentOptions,
+        BUTTON_TOOLTIP_ALIGNMENT.CENTER
+      ),
+      tooltipPosition: select(
+        'Tooltip position',
+        positionOptions,
+        BUTTON_TOOLTIP_POSITION.TOP
+      ),
+      tooltipText: text('Tooltip text', 'Icon description'),
       disabled: boolean('Disabled (disabled)', false),
       size: select('Button size (size)', sizes, null),
       href: textNullable('Link href (href)', ''),
       onClick: action('click'),
-      isExpressive: boolean('expressive (isExpressive)', false),
     }),
   },
 };
 
-export const skeleton = (args) => {
-  const { disabled, size, href, isExpressive, onClick } =
-    args?.[`${prefix}-btn-skeleton`];
+export const Secondary = (args) => {
+  const {
+    disabled,
+    href,
+    size,
+    tooltipAlignment,
+    tooltipPosition,
+    tooltipText,
+    onClick,
+  } = args?.[`${prefix}-btn`] ?? {};
   return html`
-    <cds-btn-skeleton
-      ?disabled=${disabled}
-      size=${ifDefined(size)}
-      href=${ifDefined(href || undefined)}
-      ?isExpressive="${isExpressive}"
-      @click=${onClick}>
-    </cds-btn-skeleton>
+    <cds-btn
```

Comment:
the react skeleton looks a bit different: https://react.carbondesignsystem.com/?path=/story/components-button--skeleton